arlo_server/jurisdictions.py

```diff
@@ -1,35 +1,115 @@
 from flask import jsonify
+from typing import List, Dict, Optional
+from enum import Enum
+from sqlalchemy import func
 
 from arlo_server import app
-from arlo_server.models import Election, Jurisdiction
+from arlo_server.models import (
+    Election,
+    Jurisdiction,
+    SampledBallotDraw,
+    SampledBallot,
+    Batch,
+    Round,
+    AuditBoard,
+)
 from arlo_server.auth import with_election_access, UserType
+from arlo_server.rounds import get_current_round
 from util.process_file import serialize_file, serialize_file_processing
+from util.jsonschema import JSONDict
 
 
-def serialize_jurisdiction(db_jurisdiction: Jurisdiction) -> dict:
+def serialize_jurisdiction(
+    jurisdiction: Jurisdiction, round_status: Optional[JSONDict]
+) -> JSONDict:
     return {
-        "id": db_jurisdiction.id,
-        "name": db_jurisdiction.name,
+        "id": jurisdiction.id,
+        "name": jurisdiction.name,
         "ballotManifest": {
-            "file": serialize_file(db_jurisdiction.manifest_file)
-            if db_jurisdiction.manifest_file
+            "file": serialize_file(jurisdiction.manifest_file)
+            if jurisdiction.manifest_file
             else None,
-            "processing": serialize_file_processing(db_jurisdiction.manifest_file)
-            if db_jurisdiction.manifest_file
+            "processing": serialize_file_processing(jurisdiction.manifest_file)
+            if jurisdiction.manifest_file
             else None,
-            "numBallots": db_jurisdiction.manifest_num_ballots,
-            "numBatches": db_jurisdiction.manifest_num_batches,
+            "numBallots": jurisdiction.manifest_num_ballots,
+            "numBatches": jurisdiction.manifest_num_batches,
         },
+        "currentRoundStatus": round_status,
+    }
+
+
+class JurisdictionStatus(str, Enum):
+    NotStarted = "NOT_STARTED"
+    InProgress = "IN_PROGRESS"
+    Complete = "COMPLETE"
```

Comment:
Should the keys be screaming snake case too? We do that for `UserType`, for example.

Reply:
Oh, I based this off `USState`, so I thought title case was the pattern. I've never heard the name screaming snake case before, but I love it so much that I will definitely switch to that